packages/webdriver-utils/src/util/cache.js

```diff
@@ -1,6 +1,6 @@
 
-import validations from './validations.js';
-const { Undefined } = validations;
+import utils from '@percy/sdk-utils';
+const Undefined = utils.Undefined;
```

Comment:
```suggestion
import {Undefined} from '@percy/sdk-utils';
```

Reply:
This needs change in `package.json`, we can pick this as in separate task to reconcile this.

Reply:
as discussed

```
const {Undefined} = utils;
```